README.md

```diff
@@ -0,0 +1,108 @@
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
+
+
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <!--
+  <a href="https://github.com//majkrzak/kot">
+    <img src="images/logo.png" alt="Logo" width="80" height="80">
+  </a>
+  -->
+
+  <h3 align="center">Kot</h3>
+
+  <p align="center">
+    Packing tool made simple
+    <br />
+    <br />
+    <a href="https://github.com/majkrzak/kot/issues">Report Bug</a>
+  </p>
+</p>
+
+
+
+<!-- TABLE OF CONTENTS -->
+## Table of Contents
+
+* [About the Project](#about-the-project)
+* [Getting Started](#getting-started)
+  * [Installation](#installation)
+* [Usage](#usage)
+* [Contributing](#contributing)
+* [License](#license)
+
+
+
+## About The Project
```

Comment:
Maybe `Rationale` and something about building Kotlin projects in as simple as possible way without gradle magic and bolierpaltes